velero-plugins/build/restore.go

```diff
@@ -55,10 +55,10 @@ func (p *RestorePlugin) Execute(input *velero.RestoreItemActionExecuteInput) (*v
 	}
 	// Skip if not internal build
 	name := build.Spec.Strategy.SourceStrategy.From.Name
-	if !strings.HasPrefix(name, build.Annotations[common.BackupRegistryHostname]) {
+	if !common.HasImageRefPrefix(name, build.Annotations[common.BackupRegistryHostname]) {
 		// Does not have internal registry hostname, skip
-		err = fmt.Errorf("build is not from internal image, skipping")
-		return nil, err
+		p.Log.Errorf("build is not from internal image, skipping")
+		return velero.NewRestoreItemActionExecuteOutput(input.Item), nil
```

Comment:
+1 i like this change. I think it makes sense to simply skip it and not return an error. Thanks for catching that.